Dockers/Dockerfile-lore-machine

```diff
@@ -25,11 +31,13 @@ RUN apt-get update && apt-get install -y \
 	python3.9 \
 	make \
     && rm -rf /var/lib/apt/lists/* && \
-	git clone --depth 1 https://github.com/The-Overlore/lore-machine.git && \
+	git clone --depth 1 https://github.com/The-Overlore/lore-machine.git -b litefs-world-db && \
```

Comment:
temporarily pulling my working branch to make debugging easier, will remove when things are sturdy/lined up